### DISCUSSION TITLE
Leetcode Quickadd Script

### DISCUSSION THREAD

**Discussion by zimmshane** [chhoumann/quickadd#711] [Show and tell]
Hi!
I recently made a script that pulls LeetCode data for a specific problem and converts it into a pretty markdown format for obsidian. Hopefully someone else can get some use out of it :)
More info on the scripts's [Github page](https://github.com/zimmshane/leetcode-puller-obsidian)
Here's a preview:
![autogen-problem-screenshot](https://github.com/chhoumann/quickadd/assets/80292594/8a2c4ef9-7fe3-4556-8ca6-77f77391ce5a)

**Comment by chhoumann** [comment]
This looks awesome!

**Reply by zimmshane** [reply]
Thank you!
